crates/modelardb_server/tests/integration_test.rs

```diff
@@ -680,15 +680,26 @@ fn test_can_collect_metrics() {
     let values_array = modelardb_common::array!(metrics, 2, ListArray);
 
     // The used_uncompressed_memory metric should record the change when ingesting and when flushing.
-    // 786432 is modelardb_common::test::UNCOMPRESSED_BUFFER_SIZE.
+    let uncompressed_buffer_size = test::UNCOMPRESSED_BUFFER_SIZE as u32;
     assert_eq!(
         values_array
             .value(0)
             .as_any()
             .downcast_ref::<UInt32Array>()
             .unwrap()
             .values(),
-        &[786432, 1572864, 2359296, 3145728, 3932160, 3145728, 2359296, 1572864, 786432, 0]
+        &[
+            uncompressed_buffer_size,
+            uncompressed_buffer_size * 2,
+            uncompressed_buffer_size * 3,
+            uncompressed_buffer_size * 4,
+            uncompressed_buffer_size * 5,
+            uncompressed_buffer_size * 4,
+            uncompressed_buffer_size * 3,
+            uncompressed_buffer_size * 2,
+            uncompressed_buffer_size,
+            0
```

Comment:
Maybe `uncompressed_buffer_size * 0` to try and make it more clear why it is zero.

Reply:
Fixed.

Reply:
Sorry, I was not aware that Clippy warns about multiplication by zero.